File: BruteForce.py

def bruteForceMethod(text, pattern):
    for i in range(len(text)):
        for j in range(len(pattern)):
            if i + j >= len(text):
                break
            if text[i+j] != pattern[j]:
                break
        else: 
            return True
    return False


#print(bruteForceMethod("Hola me llamo bradly", "bradly"))

def bruteForce(text, pattern):
    ls = len(text)
    lp = len(pattern)
    max = (ls - lp +1)

    for i in range(max-1):
        flag = True
        j = 0
        while(j < lp and flag == True):
            if(pattern[j] != text[j+i-1]):
                flag = False
            else:
                flag == True
            j += 1
        if flag == True:
            return i
    return 0



pattern = "culear"
text= "Duran me va a culear mañana"

print(bruteForce(text,pattern))

File: BruteForce.js
function bruteForce(p,s){
    ls = s.length
    lp = p.length
    max = (ls - lp +1)

    for(let i = 0; i < max ; i++){
        let flag = true
        for(let j = 0; j < lp && flag == true; j++ ){
            if(p[j] != s[j+i-1]){
                flag = false
            }
        }if(flag == true){
            return i
        }
    }
    return 0
}

let p = "as"
let s = "hola mucho gusto"

console.log(bruteForce(p,s));